### ISSUE TITLE
`@stylistic/declaration-block-semicolon-space-before` takes the semicolon into an inline comment under `postcss-less` where the comment holds `!important`

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#211]
Found by the review of [#207](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/207), whose fix took the guard of [#117](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/117) as its model and found the gap in the model.

### What minimal example or steps are needed to reproduce the bug?

```less
a {
  color: red // c !important
  ;
}
```

Less reads `!important` here as the text of the comment, and compiles the declaration to `color: red;` with no flag.

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "customSyntax": "postcss-less",
  "rules": {
    "@stylistic/declaration-block-semicolon-space-before": "never"
  }
}
```

The `always` options do the same with a space in place of nothing.

### How did you run Stylelint?

```shell
stylelint "**/*.less" --config myconfig.json --fix
```

### What did you expect to happen?

The problem is reported and the code left as it is, as [#117](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/117) has it for `color: red // c` followed by `;`: an inline comment ends only with a line break, so there is nowhere to put the semicolon that the comment would not swallow.

### What actually happened?

The semicolon is pulled up onto the comment's line and commented out, and nothing is reported:

```less
a {
  color: red // c !important;
}
```

Under `postcss-scss` the same file is reported and left alone.

### What is the cause?

`postcss-less` reads a flag out of the text of a `//` comment: the declaration comes back with a value of `red // c`, `important` set, and a raw of ` !important` followed by the whitespace. The guard asks whether the text the fix trims ends with an inline comment — the raw of the flag where the declaration carries one — and that raw alone holds no comment, so the fix goes through. Only the value and the raw together show the comment running on to the semicolon; [#207](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/207) asks the two together in `declaration-block-semicolon-newline-before`, and this rule wants the same.
